### PR TITLE
Add USAGE

### PR DESCRIPTION
**Pull request by konsumer**
This expands the usage, ad puts it in a separate file from README. Related to #7

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+> **TODO**: Add more here
+
+You can see a nice demo of usage, using godot, [here](https://youtu.be/mmUxl46h24M)
+
+### API
+
+All lines can be either `<CHARACTER>: message`, where `<CHARACTER>` is the name of the person talking. Commands (`if` / `do`) are run in the context of you rscene/class, so use your Godot scene to define variables and action-methods (for `do`.) Choices go in the bottom-section, 
+
+- `[if CONDITION]` - Test for a condition before calling the dialog
+- `[do ACTION (param, param, param...)]` - Run method or assign directly (like `[do cool=1]`])
```
Comment (nathanhoad) [inline USAGE.md:48]:
```suggestion
- `[do ACTION param param param...]` - Run method or assign directly (like `[do cool=1]`])
```

File: USAGE.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+SayWhat has it's own [DSL](https://en.wikipedia.org/wiki/Domain-specific_language), that is unconcered with how you will be using it. This means with one editor & language, you can edit the dialogs for any game engine/framework (aslong as you setup a loader on tat end.)
+
+### Bindings
+
+SayWHat has bindings for a few games/frameworks. You will still need to set up how it is displayed, but these will get you started 
+
+- [Godot](https://github.com/nathanhoad/saywhat_godot)
+
+> **TODO**: Add more here
```
Comment (nathanhoad) [inline USAGE.md:39]: At the moment the only public bindings are for Godot.

**Comment by konsumer** [comment]
Additionally, maybe we should put a note about `saywhat-parse` in there, for people that want to use it in code or from CLI, but I'd also like to finish the linter, which might take a little longer.

**Comment by konsumer** [comment]
ALso, there is a typo:
> if your `GODOT_SCENE.has_met_character` is truthy,

should be "not truthy"

**Review by nathanhoad — CHANGES_REQUESTED** [review]
Apart from the syntax change it looks good to me so far.

**Comment by konsumer** [comment]
Ok, I think I got your suggestions in there.

**Pull request merged**